crds/v1alpha1/src/main/crds/analysistemplates.argoproj

```diff
@@ -2779,7 +2779,7 @@ spec:
                               type: object
                           required:
                           - address
-                          - application
+                          - applicationConfig
```

Comment:
This definitely doesn't seem right. I don't think that CRDs changed since last time I pulled them.